### PR TITLE
use 'create token' command to generate token

### PR DESCRIPTION
**Pull request by carlin-q-scott**
The secret that was referenced in these instructions did not exist for me. But I had used the `create token` command before to do this same thing for the kubernetes-dashboard, so I figured it would work for kiali, and it did!

### CODE REVIEW

**Comment by israel-hdez** [comment]
@carlin-q-scott Which Kubernetes version are you using?

**Comment by carlin-q-scott** [comment]
> kubectl version
Client Version: v1.25.0
Kustomize Version: v4.5.7
Server Version: v1.25.0

**Comment by jmazzitelli** [comment]
Ah yes. The latest versions of k8s no longer create this secret for you in the SA.

We might have to keep the old docs, too, and just mention somethign like "If you are using k8s 1.24+ the secret is no longer generated in the SA itself... blah blah blah".

Here's an article someone wrote up about it: https://itnext.io/big-change-in-k8s-1-24-about-serviceaccounts-and-their-secrets-4b909a4af4e0

**Comment by israel-hdez** [comment]
@carlin-q-scott So, I agree with Mazz that the old docs still apply and you shouldn't replace any text. Just add as a note that for Kubernetes versions 1.24+ the token won't exist by default and it should be created with the command you found.

**Comment by jshaughn** [comment]
@carlin-q-scott , please update the PR with the feedback above, thanks.

**Comment by carlin-q-scott** [comment]
@jshaughn @israel-hdez I tried to incorporate your feedback into the PR. Please LMK if it's acceptable.

**Comment by jmazzitelli** [comment]
here's the netlify link for review: https://deploy-preview-591--kiali.netlify.app/docs/faq/authentication/#how-to-obtain-a-token-when-logging-in-via-token-auth-strategy

File: content/en/docs/FAQ/authentication.md [hunk 1]
```diff
@@ -16,7 +20,13 @@ For example, if you want to log into Kiali using Kiali's own service account, yo
 kubectl get secret -n istio-system $(kubectl get sa kiali-service-account -n istio-system -o "jsonpath={.secrets[0].name}") -o jsonpath={.data.token} | base64 -d
 ```
 
-Note that this example assumes you installed Kiali in the `istio-system` namespace.
+**For kubernetes v1.24+**
+
+You can request a short lived token for a service account by issuing the following command:
+
+> kubectl -n istio-system create token kiali-service-account
```
Comment (jmazzitelli) [inline content/en/docs/FAQ/authentication.md]: for markdown, you need to use triple-backquotes to format this command properly. So this should be this ~~(ignore the backslashes - I can't figure out how to embed the triple-backquotes inside a suggestion - it should be triple backquotes, then newline, then the kubectl command, then newline, then another triple backquote)~~:

````suggestion
```
kubectl -n istio-system create token kiali-service-account
```
````

File: content/en/docs/FAQ/authentication.md [hunk 2]
```diff
@@ -16,7 +20,13 @@ For example, if you want to log into Kiali using Kiali's own service account, yo
 kubectl get secret -n istio-system $(kubectl get sa kiali-service-account -n istio-system -o "jsonpath={.secrets[0].name}") -o jsonpath={.data.token} | base64 -d
 ```
 
-Note that this example assumes you installed Kiali in the `istio-system` namespace.
+**For kubernetes v1.24+**
```
Comment (jmazzitelli) [inline content/en/docs/FAQ/authentication.md]: Kubernetes should be capitalized: https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list

```suggestion
**For Kubernetes v1.24+**
```

File: content/en/docs/FAQ/authentication.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ description: "Questions about authentication strategy or configuration."
 
 When configuring Kiali to use the `token` auth strategy, it requires users to log into Kiali as a specific user via the user's service account token. Thus, in order to log into Kiali you must provide a valid Kubernetes token.
 
+Note that the following examples assume you installed Kiali in the `istio-system` namespace.
+
+**For kubernetes prior to v1.24**
```
Comment (jmazzitelli) [inline content/en/docs/FAQ/authentication.md]: Kubernetes should be capitalized. See: https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list

```suggestion
**For Kubernetes prior to v1.24**
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
some minor things but the content overall looks good

**Comment by israel-hdez** [comment]
@carlin-q-scott do you want to commit the other suggestions?

**Review by jmazzitelli — APPROVED** [review]

**Comment by jmazzitelli** [comment]
I committed the two other changes (needed to capitalize `Kubernetes`). This is ready once it goes green.

**Pull request merged**